### DISCUSSION TITLE
How to pass parameters to --query

### DISCUSSION THREAD

**Discussion by AkhilaGarlapally** [aws/aws-cli#7308] [Q&A] [answered]
Hi,

1. When I try this command
 
``` aws gamelift list-builds --region us-west-2 --query 'Builds[?starts_with(Name, ` "mybuild" `) == `true`].BuildId' --output text```

**output**:  build-xxxx

2. When I try this command
```
$var=mybuild
aws gamelift list-builds --region us-west-2 --query 'Builds[?starts_with(Name, `"$${var}"`) == `true`].BuildId' --output text
```
**output** No Output
**ExpectedOutput** build-xxxx

Can you please help me how can I make second attempt success

**Comment by aBurmeseDev** [comment, marked as answer]
Hi @AkhilaGarlapally, thanks for reaching out.

AWS uses [JMESPath expression](https://jmespath.org/specification.html) and it is enclosed in single quotes. In this case, you can surround the `--query` value with double quotes and then use single quotes ([raw string literals](http://jmespath.org/specification.html#raw-string-literals)) for the $mybuild var like so:

```
var=mybuild
aws gamelift list-builds --region us-west-1 --query "Builds[?starts_with(Name, '${var}')].BuildId" --output text
```
I was able to get the expected output with this command above. Please let me know if you're still having the issue.

Hope it helps,
John
